src/widgets/TextWidget.tsx

```diff
@@ -122,6 +130,7 @@ TextWidget.convertProps = (props: TextWidgetProps): TextWidgetInternalProps => {
     ...buildTextShadow(props.style ?? {}),
     ...(props.truncate ? { truncate: props.truncate } : {}),
     ...(props.maxLines ? { maxLines: props.maxLines } : {}),
+    decorationLine: props.decorationLine ?? 'none'
```

Comment:
Use this from `props.style`, and spread it like truncate/maxLines since it is optional. Don't put 'none' as a default.